src/ui/public/index_patterns/_index_pattern.js

```diff
@@ -95,9 +95,21 @@ export function IndexPatternProvider(Private, Notifier, config, kbnIndex, Promis
     return promise;
   }
 
-  function containsFieldCapabilities(fields) {
-    return _.any(fields, (field) => {
-      return _.has(field, 'aggregatable') && _.has(field, 'searchable');
+  function isFieldRefreshRequired(indexPattern) {
+    if (!indexPattern.fields) {
+      return true;
+    }
+
+    return indexPattern.fields.every(field => {
+      // added in 5.0 #8421
```

Comment:
First time I've seen this kind of comment in Kibana (but that might just be because I haven't looked at much code yet). Is there a specific reason for this here?